src/i18n/ja/ui.ts

```diff
@@ -7,12 +7,14 @@ export default UIDictionary({
 	'site.title': 'Astroドキュメント',
 	'site.description':
 		'より少ないクライアントサイドJavascriptで、より高速なWebサイトを構築できます。',
+	'site.og.imageSrc': '/default-og-image.png?v=1',
```

Comment:
これって、翻訳でもないし、各言語のui.tsに必要なんでしょうか？（`fr/ui.ts`にあるのは確認しました）

@delucis I have a question.
Is `site.og.imageSrc` needed in `ui.ts` for each language?

Reply:
自分もそこは気になりました。https://github.com/withastro/docs/blob/eb0657de3fff32d7e4550960f448ad558145a21a/src/components/HeadSEO.astro#L18 における使われ方を見るに、フォールバック用の画像として使われるようですね。現在はあまり意味がありませんが、将来言語ごとにデフォルトの画像が別々になったときのためのプレースホルダーという感じでしょうか？

---

I was a little wondering about that too. Looking at how it's used on https://github.com/withastro/docs/blob/eb0657de3fff32d7e4550960f448ad558145a21a/src/components/HeadSEO.astro#L18 it seems like the image is for fallback. I think it doesn't make much sense to include `site.og.imageSrc` right now, but maybe it's a placeholder for a future version where each language has its own default og image?

Reply:
Not required currently no! The reasons we decided to include the URL here were a) so that image URL and alt text were in the same place and easier to update together and b) in case at some point in the future we had a different image for each language.

So we could leave this out for now.